docs/getting_started/project_progress.md

```diff
@@ -28,16 +41,22 @@ For more information, see [Project Dashboard](../webapp/applications/apps_dashbo
 ![Project Dashboard](../img/apps_dashboard.png#light-mode-only)
 ![Project Dashboard](../img/apps_dashboard_dark.png#dark-mode-only)
 
-### Project Overview
+## Task Monitoring
 
-A project's **OVERVIEW** tab in the UI presents a general picture of a project: 
-* Metric Snapshot – A graphical representation of selected metric values across project tasks, offering a quick assessment of progress.
-* Task Status Tracking – When a single metric variant is selected for the snapshot, task status is color-coded (e.g., 
-Completed, Aborted, Published, Failed) for better visibility.
+ClearML task monitoring capabilities through the [`clearml.automation.Monitor`](https://github.com/clearml/clearml/blob/master/clearml/automation/monitor.py) 
+class. By extending this class, you can:
```

Comment:
@pollfly "By extending this class, you can" -> "With this class you can implement monitoring workflows such as"